README.md

```diff
@@ -61,6 +61,20 @@ Add the `Okta OIDC` dependency to your `build.gradle` file:
 implementation 'com.okta.android:oidc-androidx:1.0.14'
 ```
 
+The SDK requires Java 8 support.
+To enable, add the following following to your `build.gradle` file
```

Comment:
https://developer.android.com/studio/write/java8-support per the documentation we should show how to configure for kotlin projects as well since that appears to be different.

Reply:
updated with kotlin and reference to the java8-support